INSTRUCTIONS.md

```diff
@@ -2,66 +2,83 @@
+## Additional software for the workshop
 
-We chose the Python/Muse/MuLES combination as it provides a lot of flexibility to hackers, but at the same time is simple enough that novice users can understand what they are doing. Because of this choice, we are stuck with Windows; however, the goal of this workshop is to teach you about BCIs in general, so that you are able to apply this knowledge to the environment and tools of your choice. We won't focus much on tools here.
+Additional software is required the workshop, specifically, the following tools:
 
-### 1. Python 2 or Python 3, and packages
+* **Muse SDK**: the software development kit provided to play around with the Muse EEG headband.
+* **MuLES**: an EEG server that allows device-agnostic applications.
 
-Python is a high-level scripting language that has been widely adopted in a plethora of applications. It is free, simple to read, and has an extensive standard library. Many packages can also be downloaded online to complement its features.
+## A Installation of software for the workshop
 
-Two packages are especially useful when dealing with scientific computing (as for BCIs): [NumPy](http://www.numpy.org/) and [matplotlib](http://matplotlib.org/). NumPy allows easy manipulation of arrays and matrices (very similar to [MATLAB](http://mathesaurus.sourceforge.net/matlab-numpy.html)), which is necessary when dealing with data such as neurophysiological signals. Matplotlib is similar to MATLAB's plotting functionalities, and can be used to visualize the signals and features we are working with.
+There are many other programming languages ( C, C++, Java, Processing, etc.); a diversity of **BCI toolboxes** ([OpenVIBE](http://openvibe.inria.fr/), [BCI2000](http://www.bci2000.org/wiki/index.php/Main_Page), [BCILAB](http://sccn.ucsd.edu/wiki/BCILAB), etc.); and even **other EEG devices** (OpenBCI, Emotiv EPOC, Neurosky Mindwave, etc.).
+
+Among those, we chose the **Python( or Matlab)-Muse-MuLES** combination as it provides a lot of flexibility to hackers, but at the same time is simple enough that novice users can understand what they are doing. Because of this choice, we are stuck with Windows OS; however, the goal of this workshop is to teach you about BCIs in general, so that you are able to apply this knowledge to the environment and tools of your choice. We won't focus much on tools here.
+
+These are the steps to setup your computer
+
+**A.1.** Installing Python and required packages  
+**A.2.** Installing Muse SDK  
+**A.3.** Installing MuLES (MuSAE Lab EEG Server)  
+**A.4.** Pairing the Muse EEG headset and configuring MuLES
+**A.5.** Download the code for the workshop
+
+### A.1 Installing Python required packages
+If you will use Matlab, and it's already installed skip to next step: **A.2**
```

Comment:
I think the proper way of writing MATLAB is all in upper case: https://en.wikipedia.org/wiki/MATLAB. Also, I would reformulate the sentence as: "If you are going to use MATLAB and it's already installed on your computer, you can skip to step **A.2**."